JS: invalid imports are generated

### The problem

Currently, when Aqua is compiled to JS, it emits the following line in every .js file:
```
import { RequestFlowBuilder } from '@fluencelabs/fluence/dist/api.unstable';
```

However, it seems that at least NodeJS doesn't know how to work with that:
```
$ npm run start

> example@1.0.0 start /Users/folex/Development/fluencelabs/aqua-dht/examples/javascript
> node example.js

internal/process/esm_loader.js:74
    internalBinding('errors').triggerUncaughtException(
                              ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/folex/Development/fluencelabs/aqua-dht/examples/javascript/node_modules/@fluencelabs/fluence/dist/api.unstable' imported from /Users/folex/Development/fluencelabs/aqua-dht/examples/javascript/export.js
Did you mean to import @fluencelabs/fluence/dist/api.unstable.js?
```

### The fix
Add `.js` to the end of the file name:
```
import { RequestFlowBuilder } from '@fluencelabs/fluence/dist/api.unstable.js';
```

Version: `0.2.1-219`

--- a/backend/js/src/main/scala/aqua/backend/js/JavaScriptFile.scala
+++ b/backend/js/src/main/scala/aqua/backend/js/JavaScriptFile.scala
@@ -13,7 +13,7 @@ object JavaScriptFile {
        | * Aqua version: ${Version.version}
        | *
        | */
-       |import { RequestFlowBuilder } from '@fluencelabs/fluence/dist/api.unstable';
+       |import { RequestFlowBuilder } from '@fluencelabs/fluence/dist/api.unstable.js';
        |""".stripMargin
 
 }

